interface/modules/custom_modules/oe-module-visualehr/Readme.md

```diff
@@ -0,0 +1,26 @@
+## Visual EHR
+This is a brief readme of how to install the visual dashboard.
+
+This is a reactjs application embeded in a php module, hence this installation is in two steps
+
+1. ReactJs Configuration
+STEP 1: 
+In the react directory (oe-module-visualehr/react), headover to the package.json file and change the homepage and proxy values to the settings that applies to you. for example
+`"homepage": "http://localhost/openemr-7/interface/modules/custom_modules/oe-module-visualehr/public/"`,
+`"proxy": "http://localhost/"`,
+
+STEP 2: run `npm i` to install node modules
+
+STEP 3: Save and build the project using `npm run build`
```

Comment:
Looks like the package.json is out of date.
![image](https://user-images.githubusercontent.com/228117/224510405-31d8ecf8-9a17-4bc1-904e-19efad2acd31.png)